app/controllers/items_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ItemsController < ApplicationController
+
+  def index
+    @items = Item.all
+  end
+
+  def show
+    @item = Item.find(params[:id])
+  end
+
+  def new
+    @merchant_id = params[:merchant_id]
+  end
+
+  def edit
+    @item_id = params[:id]
+  end
+
+  def update
+    Item.find(params[:id]).update(item_params)
+    redirect_to "/items/#{params[:id]}"
+  end
+
+  def create
+    created_params = item_params
+    created_params[:active_status] = true
```

Comment:
You can set this active_status to true here, or you can look into making this column in your database a default. You'll just have to go about validating that the column exists a little differently since it is a default. If the value is false, it would say that it doesnt exist. So you'd have to validate the inclusion of that column instead of the presence of it.